ts/src/coinbase.ts

```diff
@@ -2263,9 +2263,9 @@ export default class coinbase extends Exchange {
         };
     }
 
-    async findAccountId (code) {
-        await this.loadMarkets ();
-        await this.loadAccounts ();
+    async findAccountId (code, params = {}) {
+        await this.loadMarkets (false, params);
```

Comment:
I don't think we should provide `params` to loadMarkets since they're targeting the accounts endpoint